app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,25 @@
+class SessionsController < ApplicationController
+  skip_before_action :authenticate_user, except: :destroy
```

Comment:
Is there any need to label except destroy I think when we destroy any user it will not check authenticate_user. you're already skipping the authenticate_user method for all actions except destroy by default, you don't need to explicitly specify it with except: :destroy. You can simply remove that part.